### ISSUE TITLE
Open source and business models

### ISSUE THREAD

**Issue by ttoine** [sfosc/sfosc#25]
A few references for the "Business models" page

http://dtrace.org/blogs/bmc/2018/12/14/open-source-confronts-its-midlife-crisis
https://opensource.com/business/15/8/open-source-products-four-rules
https://blog.mozilla.org/blog/2018/05/15/whats-your-open-source-strategy-here-are-10-answers/ (read the document in pdf)

> Open source in not a business model on its own

> Successful companies use open source projects to build product that their customers will buy to solve their problems.

BTW, a short introduction: I am working for PrestaShop, an open source solution to sell online, where I am the developer advocate. I also cofounded Ubuntu Studio in the past, and participate to several open source communities. But, I am not a coder.

**Comment by adamhjk** [comment]
I struggled with having a "business models" section. While I agree that open source isn't a business model, I think in the conception of community, it certainly includes the business models used by companies in open source. Do you think we should add a reading list page? Change the phrasing?

**Comment by adamhjk** [comment]
Absolutely on a reading list, #28 is just that. 

I'll read the report and get back. One thing I like about calling it "business model" is that, right or wrong, it's what people call it in the world who aren't wonky about it. The odds someone would say "I don't like the open source archetype" you are using is pretty low. I also recognize this is a tautology (people don't use that language, so we don't use that language, so people don't use that language...)

**Comment by ttoine** [comment]
Yes, you are right, people are expecting "business model". However, it's not mandatory to use "open source business model", and instead, use a better phrasing. Examples:

- "How to create a business model on top of an open source project?"
- "In order to build a strong business model, you must identify people and companies who have money to solve their problem, and then build that for them"

Finding a way to explain that an open source project is not something you can sell directly like a product is important.

**Comment by hennevogel** [comment]
IMHO that language (business model) also forms thinking. For instance, and this might simply be an oversight, where is the *business model* in this section that is the basis of a sustainable open source project that is **not** involving capital flow?
